### PR TITLE
deprecate GOPATH references on docs 

### PR DESCRIPTION
**Pull request by tac0turtle**
- deprecate setting of gopath in docs and bump min version to go1.13

sorry for the formatting of the files, if you want I can undo the changes, just let me know
Signed-off-by: Marko Baricevic <marbar3778@yahoo.com>

- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Reviewed `Files changed` in the github PR explorer

______

For Admin Use:
- Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- Reviewers Assigned
- Squashed all commits, uses message "Merge PR #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -8,17 +8,16 @@ mainnet as either a [Full Node](./join-mainnet.md) or a
...
 
 Install `go` by following the [official docs](https://golang.org/doc/install).
-Remember to set your `$GOPATH` and `$PATH` environment variables, for example:
+Remember to set your `$PATH` environment variable, for example:
 
 ```bash
 mkdir -p $HOME/go/bin
-echo "export GOPATH=$HOME/go" >> ~/.bash_profile
-echo "export PATH=\$PATH:\$GOPATH/bin" >> ~/.bash_profile
+echo "export PATH=$PATH:/usr/local/go/bin" >> ~/.bash_profile
```
Comment (alessio) [inline docs/installation.md]:
```suggestion
echo "export PATH=$PATH:$(go env GOPATH)/bin" >> ~/.bash_profile
```

Comment (alessio) [reply docs/installation.md]: In my view, we should not teach the user how to install Go. Plus, `/usr/local/go` is not an OS-agnostic path

**Review by fedekunze — APPROVED** [review]
@marbar3778 I didn't know you spoke Korean

File: docs/installation.md (hunk 1 above)
Comment (jackzampolin) [reply docs/installation.md]: I agree @alessio, we should just link to the golang install docs: https://golang.org/doc/install

Comment (tac0turtle) [reply docs/installation.md]: this what is being done: https://github.com/cosmos/gaia/blob/master/docs/installation.md#install-go. this bash excerpt is merely showing an example to not forget to set your env variable. we can get rid of the bash section as well if we'd like

File: docs/join-testnet.md [hunk 2]
```diff
@@ -47,13 +47,14 @@ Make sure that every node has a unique `priv_validator.json`. Do not copy the `p
 Now it is time to upgrade the software:
 
 ```bash
-cd $GOPATH/src/github.com/cosmos/gaia
+git clone https://github.com/cosmos/gaia.git
+cd gaia
 git fetch --all && git checkout master
 make update_tools install
```
Comment (alessio) [inline docs/join-testnet.md]:
```suggestion
make install
```

File: docs/translations/cn/join-testnet.md [hunk 3]
```diff
@@ -45,15 +44,16 @@ gaiad unsafe-reset-all
 现在升级软件：
 
 ```bash
-cd $GOPATH/src/github.com/cosmos/cosmos-sdk
+git clone https://github.com/cosmos/cosmos-sdk.git
+cd cosmos-sdk
 git fetch --all && git checkout master
 make update_tools install
```
Comment (alessio) [inline docs/translations/cn/join-testnet.md]:
```suggestion
make install
```

File: docs/translations/kr/join-testnet.md [hunk 4]
```diff
@@ -70,13 +69,14 @@ gaiad unsafe-reset-all
 이제 소프트웨어를 업그레이드할 시간입니다:
 
 ```bash
-cd $GOPATH/src/github.com/cosmos/cosmos-sdk
+git clone https://github.com/cosmos/cosmos-sdk.git
+cd cosmos-sdk
 git fetch --all && git checkout master
 make update_tools install
```
Comment (alessio) [inline docs/translations/kr/join-testnet.md]:
```suggestion
make install
```

**Pull request merged**